lib/ansible/utils/display.py

```diff
@@ -62,7 +62,7 @@ def filter(self, record):
 if getattr(C, 'DEFAULT_LOG_PATH'):
     path = C.DEFAULT_LOG_PATH
     if path and (os.path.exists(path) and os.access(path, os.W_OK)) or os.access(os.path.dirname(path), os.W_OK):
-        logging.basicConfig(filename=path, level=logging.INFO, format='%(asctime)s p=%(user)s u=%(process)d | %(message)s')
+        logging.basicConfig(filename=path, level=logging.INFO, format='%(asctime)s p=%(process)d u=%(user)s | %(message)s')
```

Comment:
Where do you get `%(user)s`? According to the documentation that key doesn't exist: https://docs.python.org/3/library/logging.html#logrecord-attributes

Using `%(user)s` gives me the following error:
```
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib64/python3.6/logging/__init__.py", line 994, in emit
    msg = self.format(record)
  File "/usr/lib64/python3.6/logging/__init__.py", line 840, in format
    return fmt.format(record)
  File "/usr/lib64/python3.6/logging/__init__.py", line 580, in format
    s = self.formatMessage(record)
  File "/usr/lib64/python3.6/logging/__init__.py", line 549, in formatMessage
    return self._style.format(record)
  File "/usr/lib64/python3.6/logging/__init__.py", line 391, in format
    return self._fmt % record.__dict__
KeyError: 'user'
```